content/geoip/updating-databases.mdx

```diff
@@ -155,11 +158,12 @@ For example, using the permalink for the GeoIP2 City CSV database, you can issue
 a command like the following:
 
 ```cli
-curl -I 'https://download.maxmind.com/app/geoip_download?edition_id=GeoIP2-City-CSV&license_key=YOUR_LICENSE_KEY&suffix=zip'
+curl -I -L -u YOUR_ACCOUNT_ID:YOUR_LICENSE_KEY 'https://download.maxmind.com/geoip/databases/GeoIP2-City-CSV/download?suffix=zip'
```

Comment:
Maybe it would be worth adding a `wget` example as well as I assume some people will want to use that.